plugins/generator-1.18.2/forge-1.18.2/templates/item/item.java.ftl

```diff
@@ -54,9 +54,23 @@ import javax.annotation.Nullable;
 				.fireResistant()
 				</#if>
 				.rarity(Rarity.${data.rarity})
+				<#if data.isFood>
+				.food((new FoodProperties.Builder())
+				    .nutrition(${data.nutritionalValue})
+                	.saturationMod(${data.saturation}f)
+                    <#if data.isAlwaysEdible>.alwaysEat()</#if>
+                    <#if data.isMeat>.meat()</#if>
+                    .build())
+                </#if>
 		);
 	}
 
+	<#if data.animation != "eat">

```

Comment:
This will set getUseAction for all cases except for eat. In case of default animation for food or non food item, this method should not be generated